assets/js/components/setup/search-console.js

```diff
@@ -48,24 +48,29 @@ class SearchConsole extends Component {
 		this.submitPropertyEventHandler = this.submitPropertyEventHandler.bind( this );
 	}
 
-	componentDidMount() {
+	async componentDidMount() {
 		const { isAuthenticated, shouldSetup } = this.props;
 
-		if ( isAuthenticated && shouldSetup ) {
-			this.requestSearchConsoleSiteList();
+		if ( ! isAuthenticated || ! shouldSetup ) {
+			return;
 		}
-	}
 
-	shouldComponentUpdate( nextProps ) {
```

Comment:
Removed `shouldComponentUpdate` as runs `requestSearchConsoleSiteList()` and makes unnecessary requests to `matched-sites` and `insert` which have already been requested.